src/SonarQube.Client/Models/SonarQubeIssue.cs

```diff
@@ -74,6 +77,9 @@ public class SonarQubeIssue
         public IReadOnlyList<IssueFlow> Flows { get; }
 
         public string Context { get; set; }
+
+        public string CleanCodeAttribute { get; }
```

Comment:
Isn't really needed in this sprint

Reply:
I've added already removing would not make sense.

Reply:
Yeah true